application/app/views/shared/_progress_bar.html.erb

```diff
@@ -1,10 +1,18 @@
 <% progress ||= 0 %>
-<div class="progress" aria-live="polite" style="height: 20px;">
-  <div class="progress-bar progress-bar-striped progress-bar-animated bg-info"
-       role="progressbar"
-       style="width: <%= progress %>%"
-       aria-labelledby="progress-bar-label-<%= file.id %>"
-       aria-valuenow="<%= progress%>" aria-valuemin="0" aria-valuemax="100">
+<div class="progress-container d-flex flex-column justify-content-end flex-grow-1" style="max-width: 6rem;">
```

Comment:
Why the max-width?
As it is in a fixed column, with the flex-grow will use all the space available.

Reply:
because it looks more compacted and with similar width as the badges. Otherwise, It would look like the example below: 

<img width="1575" height="868" alt="image" src="https://github.com/user-attachments/assets/49291947-765b-4c1b-bf7c-37e07c94bc53" />

If you prefer the latter approach, I can remove the `max-width`